example-get-started-experiments/generate.sh

```diff
@@ -75,17 +76,17 @@ dvc pull
 
 
 cp -r $HERE/code/notebooks .
+cp -r $HERE/code/sagemaker .
 git add .
 git commit -m "Add notebook using DVCLive"
 
+sudo apt-get update && sudo apt-get install ffmpeg libsm6 libxext6 -y
 pip install -r requirements.txt --extra-index-url https://download.pytorch.org/whl/cu118
 pip install jupyter
+yolo settings datasets_dir=/workspaces/example-repos-dev/example-get-started-experiments/build/example-get-started-experiments/data
```

Comment:
so, how will it look like in docs - we'll we have to explain all this in the get started?

Reply:
No, this is only needed during generation inside codespaces because YOLO resolves the paths wrong inside `generate.sh`.
For someone cloning the repo it works without running the command